### PR TITLE
Test Plot functions

### PR DESCRIPTION
**Pull request by grillmann**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -51,6 +51,7 @@
     "papaparse": "^4.3.7",
     "pg": "^7.4.1",
     "swagger-ui-express": "^2.0.15",
+    "underscore": "^1.8.3",
```
Comment (r-sukanya) [inline package.json]: are you using underscore ?

Comment (grillmann) [reply package.json]: oh no sorry i think that was just to try something out

File: server/test/unit/app.test.js [hunk 2]
```diff
@@ -70,4 +73,131 @@ describe('POST /API/nlp', function () {
             .send({dataset: "human_resources__core_dataset", input: "Plot a histogram of age", history: []})
             .expect(200, done);
     });
-});
\ No newline at end of file
+});
+
+describe('Test Simple Plot Functions', function () {
+    it('should render ok',async function () {
```
Comment (sebmarste) [inline server/test/unit/app.test.js]: could you please give the single tests better names (better than every time `should render ok`)

File: server/test/unit/app.test.js [hunk 3]
```diff
@@ -70,4 +73,131 @@ describe('POST /API/nlp', function () {
...
+        }, (error) => {
+            reject(error);
+        })});
+        let isOver50 = false
+        result.data[0].x.forEach(value => {
+            if(value > 50){
+                isOver50 = true
+            }
+        })
+        chai.assert(!isOver50);
```
Comment (sebmarste) [inline server/test/unit/app.test.js]: what about
`chai.assert.notExists(result.data[0].x.find(value => value > 50));`

File: package.json (hunk 1 above)
Comment (sebmarste) [reply package.json]: then remove it please ;)

File: server/test/unit/app.test.js (hunk 2 above)
Comment (grillmann) [reply server/test/unit/app.test.js]: oh ofc, i thought that was just a convention

**Pull request merged**